guava/src/com/google/common/base/Equivalence.java

```diff
@@ -345,6 +345,17 @@ public static Equivalence<Object> equals() {
     return Equals.INSTANCE;
   }
 
+  /**
+   * Returns an equivalence that delegates to {@link BiPredicate#test(Object, Object)} and {@link Object#hashCode}.
+   * {@link Equivalence#equivalent} returns {@code true} if the input values match the predicate, including in the case that a and b are both null. {@link Equivalence#hash} returns
+   * {@code 0} if passed a null value.
+   *
+   * @since 31.1
```

Comment:
```suggestion
   * @since NEXT
```
(This will be replaced before release.)